defi/src/adaptors/handlers/storeAdaptorData/index.ts

```diff
@@ -267,14 +269,65 @@ export const handler2 = async (event: IStoreAdaptorDataHandlerEvent) => {
             }
           }
 
+          // Edge case: V2 adapter has yesterday data but it was updated before end of yesterday (incomplete day)
+          // Valid update window: 00:00 UTC (today) - 01:00 UTC (today) = fresh data, no refill needed
+          // If updatedAt < 00:00 UTC today (i.e., from yesterday), reload after 01:00 UTC today
+          if (!runAtCurrTime && haveYesterdayData && yesterdayDataMap.has(id2)) {
```

Comment:
was thinking, what if this logic to storeAll instead, my worry is, this script/file is already complex and the new code is applicable only to store-all task.

was thinking along the lines of, not passing today's data if we want to refill today, and similar for yesterday's data based on the checks in this PR, wdyt?